### ISSUE TITLE
Wcag21aa

### ISSUE THREAD

**Issue by karl0ss** [dequelabs/axe-core#1273]
Can someone tell me how I am able to run axe-core with wcag21a/aa rules, as I only seem to find information on running it with wcag2a/aa rules..

I have looked on the "aXe Javascript Accessibility API" and there is nothing about 21.

Thanks

**Comment by WilcoFiers** [comment]
Right you are! There's a wcag21aa tag you can use. It should be running by default though. We don't currently have WCAG 2.1 level A rules, so there is no tag for that.
